### DISCUSSION TITLE
how about rocksdb for backen storage

### DISCUSSION THREAD

**Discussion by dear-Xia** [JanusGraph/janusgraph#2240] [Ideas]
For discussion on internal implementation details of JanusGraph or proposing a [major new feature or subproject](https://docs.janusgraph.org/development/#development-decisions) use the [janusgraph-dev](https://groups.google.com/forum/#!forum/janusgraph-dev).

**Describe the feature:**
TODO

**Describe a specific use case for the feature:**
TODO

**Comment by li-boxuan** [comment]
Is there any specific reason you are interested in RocksDB as backend storage? Have you tried BerkeleyDB (https://docs.janusgraph.org/storage-backend/bdb/)?
